packages/hint-ssllabs/src/api.ts

```diff
@@ -0,0 +1,29 @@
+/* istanbul ignore file */
+
+import got from 'got';
+import {delay} from '@hint/utils';
+import { SSLLabsOptions, SSLLabsResult } from './types';
+
+const APIURL = 'https://api.ssllabs.com/api/v3/analyze';
+
+const analyzeWithRetry = async (options: SSLLabsOptions): Promise<SSLLabsResult> => {
```

Comment:
[`p-retry`](https://www.npmjs.com/package/p-retry) is pulled as a devDependency in case you want to "promote it" and have other features like `retries` and exponential backoff.

Reply:
Maybe the name is not the best one, but more than retrying, we are polling(?) until the status is ready.